src/util/object.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint "@typescript-eslint/explicit-module-boundary-types": off */
+
+// Utility function to test whether an object has own nested property
+// This is used instead of Object.prototype.hasOwnProperty() due to browser support
+export function hasProperty(obj: any, prop: string): boolean {
```

Comment:
out of curiosity, wouldn't it be better to use js-doc blocks? i.e.:
```javascript
/**
 * Utility function to test whether an object has own nested property
 * This is used instead of Object.prototype.hasOwnProperty() due to browser support
 **/
export function hasProperty(obj: any, prop: string): boolean {
```